### PR TITLE
[CMSP-791] Document changes to stderr in WP-CLI

### PR DESCRIPTION
**Pull request by pwtyler**
Closes: https://getpantheon.atlassian.net/browse/CMSP-791

## Summary
Documents upcoming changes to WP-CLI's stderr changes. Language around current/future state should be adjusted when the change is rolled out, and then this section removed in 2-3 months.

[WP-CLI Troubleshooting](https://docs.pantheon.io/guides/wp-cli/wp-cli-troubleshoot)

**Release**:
- [x] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/old-path/` => `/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Review by rachelwhitton — APPROVED** [review]

File: source/content/guides/wp-cli/07-wp-cli-troubleshoot.md [hunk 1]
```diff
@@ -58,7 +58,64 @@ ini_set('arg_separator.output', '&');
...
+ [notice] Command: <site>.<env> -- wp config get table_prefix [Exit: 0]
+$ echo $PREFIX
+
+Notice: A Notice. in phar:///opt/pantheon/wpcli/wp-cli-2.8.1.phar/vendor/wp-cli/config-command/src/Config_Command.php(444) : eval()'d code on line 78
+
+Warning: A Warning. in phar:///opt/pantheon/wpcli/wp-cli-2.8.1.phar/vendor/wp-cli/config-command/src/Config_Command.php(444) : eval()'d code on line 79
+wp_
+```
+
+### Starting January 15th
```
Comment (scottbuscemi) [inline source/content/guides/wp-cli/07-wp-cli-troubleshoot.md:91]: @pwtyler could we add "2024" for all instances of the date? TY :)

**Comment by pwtyler** [comment]
@rachelwhitton, @scottbuscemi, Happy new year! Anything further we need on this, or good to go?

**Comment by scottbuscemi** [comment]
Ah, please publish!

**Comment by scottbuscemi** [comment]
@jazzsequence pls merge ty

**Pull request merged**